cli/cook/subcommands/config.py

```diff
@@ -0,0 +1,118 @@
+from cook.util import print_info
+
+from cook import configuration, colors
+from cook.configuration import load_config
+
+
+def get_in(dct, keys):
+    """Gets the value in dct at the nested path indicated by keys"""
+    for key in keys:
+        if not isinstance(dct, dict):
+            return None
+        try:
+            dct = dct[key]
+        except KeyError:
+            return None
+    return dct
+
+
+def set_in(dct, keys, value):
+    """Sets the value in dct at the nested path indicated by keys"""
+    for key in keys[:-1]:
+        if key not in dct:
+            dct[key] = {}
+        dct = dct[key]
```

Comment:
`dct = dct.get(key, {})`

Reply:
I actually need to set `dct[key]` to `{}` to create the nested map if it's not present